src/names.ts

```diff
@@ -1,3 +1,3 @@
 export const ARRAY_META_KEY = ".zarray";
 export const GROUP_META_KEY = ".zgroup";
-export const ATTRS_META_KEY = ".zattributes";
+export const ATTRS_META_KEY = ".zattrs";
```

Comment:
Am I missing something here? All our data generated using zarr-python uses `.zattrs` for the node metadata. I've seen `.zattributes` used as a special key for [consolidated metadata](https://zarr.readthedocs.io/en/stable/api/convenience.html#zarr.convenience.consolidate_metadata) but not node metadata.

Reply:
I would have thought `.zmetadata` was the consolidated key.

Reply:
I think this is a bug and why we haven't been able to use the hierarchy core in zarr.js for traversing a remote store with metadata (eg ome-zarr).